README.md

```diff
@@ -69,7 +74,11 @@ sbatch -t 02:00:00 --mem=8G -J pharmgkb-evidence -o pharmgkb-evidence.out -e pha
 
 Update the [metrics spreadsheet](https://docs.google.com/spreadsheets/d/1Vhdajf_Aps0z9_bbHshbQthl7lHsQLxEnNBKKHUr-GE/edit#gid=0) based on the output of the pipeline.
 
-The evidence string file (`evidence.json`) must be uploaded to the [Open Targets Google Cloud Storage](https://console.cloud.google.com/storage/browser/otar012-eva/) to the **`pharmacogenomics`** folder and be named in the format `cttv012-[yyyy]-[mm]-[dd].json.gz` (e.g. `cttv012-2020-10-21.json.gz`).
+The evidence string file (`evidence.json`) must be uploaded to the [Open Targets Google Cloud Storage](https://console.cloud.google.com/storage/browser/otar012-eva/).
+To do this, run the following:
+```shell
+${CODE_ROOT}/env/bin/upload_to_gcloud.py --input-file evidence.json --destination-folder pharmacogenomics
```

Comment:
Same comment as in Clinvar the evidence string should be compressed ahead if the upload
```suggestion
gzip evidence.json
${CODE_ROOT}/env/bin/upload_to_gcloud.py --input-file evidence.json --destination-folder pharmacogenomics
```